ckanext/datastore/blueprint.py

```diff
@@ -67,30 +74,127 @@ def dump(resource_id: str):
             )
         )
 
-    response = make_response()
-    response.headers[u'content-type'] = u'application/octet-stream'
+    fmt = data[u'format']
```

Comment:
Let's drop all the `u` prefixes in new code :pray:

Reply:
Done!